### DISCUSSION TITLE
I made Windows 7 in Power Point

### DISCUSSION THREAD

**Discussion by DegroidTEAM** [community/community#148054] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

hello everyone, I'm new to github and I want to talk about my project as a Windows 7 powerpoint edition, which you can download from this link-https://github.com/gameworldNEW/windows-7-power-point-edition

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @gameworldNEW!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
